packages/beauty-portal-theme/src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {FunctionComponent, useRef, useState} from 'react';
+import RichText from '../RichText';
+import './styles.scss';
+import Chevron from '../Chevron';
+
+const Accordion: FunctionComponent<AccordionItemInterface> = accordionItem => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [height, setHeight] = useState<string>('0px');
+
+  const content = useRef(null);
+
+  const toggle = () => {
+    setIsOpen(!isOpen);
+    // @ts-ignore
```

Comment:
should this be removed?

Reply:
Ignored because the React Typescript definitions say this could be null, so we ignore to prevent a warning in IDEs.